app/containers/ProfilePage/index.js

```diff
@@ -1,4 +1,6 @@
 import React, { useEffect } from 'react';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
```

Comment:
Always add dependencies to package.json file.

Reply:
I added the dependency locally but just didn't push  the package.json file ,also for now Tabs are not needed ,I removed the tabs I added but forgot to remove the import statements